pack3d/model.go

```diff
@@ -175,30 +175,29 @@ func (m *Model) ValidBound(i int, singleStlSize []fauxgl.Vector, frameSize fauxg
 	var points []fauxgl.Vector
 	var point fauxgl.Vector
 
-	item := m.Items[i]
-	rotation := Rotations[item.Rotation]
+	transformation := m.Transformation()[i]
 	size := singleStlSize[i]
 
-	points = append(points,fauxgl.V(0.0, 0.0, 0.0))
-	points = append(points,fauxgl.V(size.X, 0.0, 0.0))
-	points = append(points,fauxgl.V(0.0, size.Y, 0.0))
-	points = append(points,fauxgl.V(0.0, 0.0, size.Z))
-	points = append(points,fauxgl.V(size.X, size.Y, 0.0))
-	points = append(points,fauxgl.V(size.X, 0.0, size.Z))
-	points = append(points,fauxgl.V(0.0, size.Y, size.Z))
-	points = append(points,size)
+	points = append(points,fauxgl.V(size.X/2, size.Y/2, size.Z/2))
```

Comment:
// Rotate around the center of volume while checking if rotation is valid. (do not rotate around origin)